qadence/states.py

```diff
@@ -8,10 +8,10 @@
 from torch import Tensor, concat
 from torch.distributions import Categorical, Distribution
 
-from qadence.blocks import ChainBlock, KronBlock, PrimitiveBlock, chain, kron
+from qadence.blocks import ChainBlock, KronBlock, PrimitiveBlock
 from qadence.circuit import QuantumCircuit
 from qadence.execution import run
-from qadence.operations import CNOT, RX, RY, RZ, H, I, X
+from qadence.operations import CNOT, RX, RY, RZ, H, I, X, chain, kron
```

Comment:
Same as above, the import looked more correct before.

Reply:
Fixed.